classes/codelist/ONIXCodelistItemDAO.php

```diff
@@ -101,13 +108,13 @@ public function _getCache(?string $locale = null): array
     public function getFilename(string $locale): string
     {
         $masterLocale = LocaleInterface::DEFAULT_LOCALE;
-        $localizedFile = "locale/{$locale}/ONIX_BookProduct_CodeLists.xsd";
+        $localizedFile = "locale/{$locale}/ONIX_BookProduct_CodeLists.xml";
```

Comment:
My system could not find the file because the file names contain small letter 'l' in 'Codelists'. So maybe to change here and below...